ajax/libs/vue-functional-calendar/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "vue-functional-calendar",
+  "title": "Vue.js Functional Calendar",
+  "description": "Lightweight, high-performance calendar component based on Vue.js",
+  "keywords": [
+    "vue",
+    "calendar",
+    "component",
+    "vue-component",
+    "functional",
+    "date picker",
+    "date range",
+    "range",
+    "multiple calendars",
+    "multiple",
+    "modal calendar",
+    "modal",
+    "inline",
+    "date",
+    "change month",
+    "change year",
+    "picker",
+    "day",
+    "month",
+    "year",
+    "time",
+    "add"
+  ],
+  "homepage": "https://github.com/ManukMinasyan/vue-functional-calendar",
+  "license": "MIT",
+  "author": "Manuk Minasyan <manuk.minasyan1@gmail.com> (https://github.com/ManukMinasyan)",
+  "filename": "FunctionalCalendar.umd.js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "npmName": "vue-functional-calendar",
```

Comment:
Your NPM package does not contain a `dist` dir:

```text
node_modules/vue-functional-calendar
├── LICENSE
├── README.md
├── _config.yml
├── babel.config.js
├── index.html
├── index.js
├── package.json
├── public
│   └── demo.png
├── src
│   ├── assets
│   │   ├── icons
│   │   │   ├── angle-down-solid.svg
│   │   │   └── angle-up-solid.svg
│   │   ├── js
│   │   │   └── helpCalendar.js
│   │   └── scss
│   │       ├── calendar.scss
│   │       └── variables.scss
│   ├── components
│   │   ├── FunctionalCalendar.vue
│   │   ├── MonthYearPicker.vue
│   │   └── TimePicker.vue
│   ├── mixins
│   │   └── propsAndData.js
│   └── utils
│       ├── EventBus.js
│       └── helpers.js
└── vue.config.js

9 directories, 20 files
```

Reply:
Done

Reply:
`2.8.72` contains no `dist` dir still.

Reply:
` "build": "vue-cli-service build --target lib --dest dist --name FunctionalCalendar src/components/FunctionalCalendar.vue",
`

how not to contain?

Reply:
Are you running build before you publish to NPM? We don't have any ability to run build on the CDN, the files must be present in the NPM package.

Reply:
Please look now 2,8,73